drivers/flash/Kconfig.andes

```diff
@@ -56,3 +77,12 @@ config FLASH_ANDES_QSPI_LAYOUT_PAGE_SIZE
 	  sector size.
 
 endif
+
+if FLASH_ANDES_QSPI_XIP
+
+config FLASH_ANDES_QSPI_XIP_COUNT_REGS
+	bool "Read flash parameters at runtime"
```

Comment:
This prompt seems incorrect. This Kconfig option is used to control how registers are accessed.

Reply:
Done.